core/src/main/resources/hudson/PluginManager/table.jelly

```diff
@@ -147,6 +147,7 @@ THE SOFTWARE.
                 <span style="margin-left:2em;"></span>
               </j:if>
               <f:submit value="${%Download now and install after restart}" />
+              <st:include page="check.jelly"/>
```

Comment:
no, installed doesn't use the table tag
https://github.com/jenkinsci/jenkins/blob/7cdb85d09c0a709c59694de7b009cb2d0117578b/core/src/main/resources/hudson/PluginManager/installed.jelly#L138-138